src/plugins/crypto.h

```diff
@@ -6,6 +6,10 @@
 
 #define BD_CRYPTO_LUKS_METADATA_SIZE (2 MiB)
 
+#define BD_CRYPTO_CHI_SQUARE_LOWER_LIMIT 136
+#define BD_CRYPTO_CHI_SQUARE_UPPER_LIMIT 426
+#define BD_CRYPTO_CHI_SQUARE_BYTES_TO_CHECK 512
```

Comment:
This looks like something that might be better as parameters for `bd_crypto_device_seems_encrypted`. I don't know if there is a change that these will change in feature, but it might be better to not have to recompile libblockdev just because you want to change these limits.

Reply:
Mmh, I don't really see the use case for changing these values - especially not in the higher level applications which will use this function. Why do you think they should be changeable without recompiling?

Reply:
I don't know, but the limits just look like something that users of libblockdev might want to change eventually. But if you think this isn't something that makes sense to change from e.g. udisks, we can keep it this way.

Reply:
I don't see why these values should be changed from udisks. We explained at https://tails.boum.org/blueprint/veracrypt/#detection why, with these values, the chi-squared test should only produce false negatives with negligible probability (1 in 10 billion). And devices with (non-encrypted) filesystem headers will never pass the test. The only reason I can think of to change these values is to handle strange filesystems, which don't have their header at least partly in the first 512 Bytes (I don't know of any such filesystem), but then I think we should handle this in libblockdev and not udisks or similar.

Reply:
Ok, you convinced me, we can keep it this way.